cmd/clairctl/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+- 1.5
+install:
+- go get -d -v ./...
+- go build -ldflags "-X github.com/coreos/clair/cmd/clairctl/cmd.version=$(cat VERSION)" -v
```

Comment:
What about using the git sha instead? See http://www.atatus.com/blog/golang-auto-build-versioning/